addons/sale_project/models/project.py

```diff
@@ -901,6 +901,16 @@ def _inverse_partner_id(self):
             if task.sale_order_id and task.partner_id.commercial_partner_id not in consistent_partners:
                 task.sale_order_id = task.sale_line_id = False
 
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
```

Comment:
No need to make that method public, it can be private since we don't need to call it with an rpc

```suggestion
    def _onchange_partner_id(self):
```

Reply:
Done, will keep that in mind from next time :smile: